src/index.html

```diff
@@ -11,8 +11,58 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), stylesheets with a .scss extension should be linked as regular CSS files. Please change the `href` attribute to point to a `.css` file instead of a `.scss` file.